packages/application/src/index.ts

```diff
@@ -59,6 +59,14 @@ export interface IPlugin<T extends Application, U> {
    */
   autoStart?: boolean;
```

Comment:
Could we use `autoStart?: boolean | 'defer'` instead of introducing another flag? This will be consistent with the actual three available state.

Reply:
Sure, it should work without breaking the current API.